src/onyx/information_model.cljc

```diff
@@ -380,18 +380,45 @@
                                :type :function
                                :optional? true
                                :added "0.8.0"}
-            :aggregation/fn {:doc "Fn (state, window, segment) to generate a serializable state machine update."
-                             :type :function
-                             :optional? false
-                             :added "0.8.0"}
-            :aggregation/apply-state-update {:doc "Fn (state, entry) to apply state machine update entry to a state."
+            :aggregation/create-state-update {:doc "Fn (window, state, segment) to generate a serializable state machine update."
+                                              :type :function
+                                              :optional? false
+                                              :added "0.8.0"}
```

Comment:
These were present in 0.8.0. You're right about the others though, I'm not sure how I missed them.

> On 22 Feb 2016, at 3:02 AM, Michael Drogalis <notifications@github.com> wrote:
> 
> In src/onyx/information_model.cljc:
> 
> > @@ -380,18 +380,45 @@
> >                                 :type :function
> >                                 :optional? true
> >                                 :added "0.8.0"}
> > -            :aggregation/fn {:doc "Fn (state, window, segment) to generate a serializable state machine update."
> > -                             :type :function
> > -                             :optional? false
> > -                             :added "0.8.0"}
> > -            :aggregation/apply-state-update {:doc "Fn (state, entry) to apply state machine update entry to a state."
> > +            :aggregation/create-state-update {:doc "Fn (window, state, segment) to generate a serializable state machine update."
> > +                                              :type :function
> > +                                              :optional? false
> > +                                              :added "0.8.0"}
> Let's bump the version to 0.9.0.
> 
> —
> Reply to this email directly or view it on GitHub.
>